fenestrate/fenestrate.py

```diff
@@ -167,8 +207,10 @@ def available_windows_between(
     and subtracts the exclusions, decorating any inclusions' data with the
     exclusions that affect it.
     """
-    all_dates = arrow.Arrow.range(
-        "day", from_time.shift(days=-1).datetime, to_time.datetime
+    # all_dates turns out to be a generator, so since we iterate over it twice,
+    # just make it a list
+    all_dates = list(
```

Comment:
This was causing a defect where the exclusions would iterate over an empty generator since it had been run through, therefore nothing was actually excluded.  We don't have any exclusion windows in Forge yet, so this hasn't been a problem yet, but hey.